src/plugins/advanced_settings/public/management_app/components/advanced_settings_voice_announcement/advanced_settings_voice_announcement.tsx

```diff
@@ -66,7 +67,14 @@ export class AdvancedSettingsVoiceAnnouncement extends Component<Props> {
     const filteredOptions = [...filteredSections];
     return (
       <EuiScreenReaderOnly>
-        <div role="region" aria-live="polite" aria-label="Advanced Settings results info">
+        <>
+          <div
+            role="region"
+            aria-live="polite"
+            aria-label={i18n.translate('advancedSettings.voiceAnnouncement.ariaLabel', {
+              defaultMessage: 'Advanced Settings results info',
+            })}
+          />
```

Comment:
```suggestion
        <div
          role="region"
          aria-live="polite"
          aria-label={i18n.translate('advancedSettings.voiceAnnouncement.ariaLabel', {
            defaultMessage: 'Advanced Settings results info',
          })}
        >
```

I think this mistakenly became a self closing `div` (the `/>` on line 77, instead of `>`) which caused the problems with the problem with children of `EuiScreenReaderOnly` requiring the `Fragment` (`<>`) to wrap the whole contents. 

Removing the self closing tag will fix the markup and all you to remove the `Fragment` as well.

Reply:
addressed